playground/infrastructure/helper.py

```diff
@@ -81,21 +84,18 @@ def find_examples(work_dir: str, categories_path: str) -> List[Example]:
     return examples
 
 
-def get_statuses(examples: List[Example]):
+async def get_statuses(examples: List[Example]):
     """
-    Receive statuses for examples and update example.status and example.pipeline_id
-
-    Use client to send requests to the backend:
-    1. Start code processing.
-    2. Ping the backend while status is STATUS_VALIDATING/STATUS_PREPARING/STATUS_COMPILING/STATUS_EXECUTING
-    Update example.pipeline_id with resulting pipelineId.
-    Update example.status with resulting status.
+    Receive status and update example.status and example.pipeline_id for each example
 
     Args:
-        examples: beam examples for processing and updating statuses.
+        examples: beam examples for processing and updating statuses and pipeline_id values.
     """
-    # TODO [BEAM-13267] Implement
-    pass
+    tasks = []
+    client = GRPCClient()
+    for example in examples:
+        tasks.append(_update_example_status(example, client))
+    await asyncio.gather(*tasks)
```

Comment:
We will need to check this before deployment to production that it doesn't ddos and stop our backend which serves it.